nixos/modules/virtualisation/google-compute-image.nix

```diff
@@ -44,10 +44,22 @@ in
         GZIP compression level of the resulting disk image (1-9).
       '';
     };
+
+    virtualisation.googleComputeImage.efi = mkOption {
+      type = types.bool;
+      default = true;
```

Comment:
Isn't this a breaking change? Like, someone has NixOS setup with non-EFI, they update the `nixpkgs` and suddenly everything is broken?

I would set this to `false` by default, unless there is a strong argument against.

Reply:
Updated